BlueDBOnDisk/src/main/java/org/bluedb/disk/file/FileUtils.java

```diff
@@ -45,10 +48,25 @@ protected FileUtils() {}  // just to get test coverage to 100%
 		return getFolderContentsExcludingTempFiles(path.toFile(), endsWithSuffix);
 	}
 
+	public static DirectoryStream<Path> getFolderContentsExcludingTempFilesAsStream(Path path, String suffix) throws IOException {
+		FileFilter endsWithSuffix = (f) -> f.toPath().toString().endsWith(suffix);
+		return getFolderContentsExcludingTempFilesAsStream(path.toFile(), endsWithSuffix);
+	}
+
+	private static DirectoryStream<Path> getFolderContentsExcludingTempFilesAsStream(File folder, FileFilter filter) throws IOException {

```

Comment:
I'd probably put this method after the next one so that it reads more like a newspaper. Both methods that use it are listed before, then you see the details below both of them. Not a big deal though and maybe not everyone's preference.

Reply:
Good call. Moved